### PR TITLE
APIv4 - Code cleanup & improve links to @see annotations in Explorer

### PR DESCRIPTION
**Pull request by colemanw**
Overview
----------------------------------------
Minor code cleanup in a few APIv4 classes, and improve rendering of links to `@see` annoations in the APIv4 explorer help panel.

Before/After
------------
Links for e.g. `Permission::get` in the APIv4 explorer were not correct (the ones at the bottom under **See:**). Now they are at least more correct.

image

### CODE REVIEW

File: Civi/Api4/Permission.php [hunk 1]
```diff
@@ -48,39 +49,33 @@ public static function getFields($checkPermissions = TRUE) {
         [
           'name' => 'group',
           'title' => 'Group',
-          'required' => TRUE,
```
Comment (eileenmcnaughton) [inline Civi/Api4/Permission.php:51]: What is the reason for these removals?

**Comment by eileenmcnaughton** [comment]
This does what it says on the box - I just had the one question about the removed 'required' lines

**Comment by eileenmcnaughton** [comment]
Hmm I hammered the UI a bit more on the 'before' & I can see that the required fields are which fields are 'required' don't actually make that much sense for the get action and don't seem to be enforced anywhere so I guess that's why - merging

**Comment by colemanw** [comment]
Thanks @eileenmcnaughton.
That's right, `required` is only enforced for `Create`.

**Pull request merged**